charts/rancher-logging/0.0.1/questions.yml

```diff
@@ -1,4 +1,6 @@
+rancher_min_version: 2.3.0-rc1
```

Comment:
Same question here, I think logging goes back further. Guess neither really matter since 2.3.0 is last supported version.

Reply:
I chose 2.3.0-rc1 for these because 2.3 is the oldest rancher version we still support in some way (security stuff only afaik), but I can change them to something even older if you prefer.

Reply:
Looks like its from 2.2, but we'll keep 2.3 since they should have upgraded by now. Same for DNS.